omod/src/main/webapp/fragments/clinicalNotes.gsp

```diff
@@ -850,6 +850,135 @@ jq(function(){
+	        source: function( request, response ) {
+	          jq.getJSON('${ ui.actionLink("patientdashboardapp", "ClinicalNotes", "getDrugs") }',
+	            {
+	              q: request.term
+	            }
+	          ).success(function(data) {
+	            var results = [];
+	            for (var i in data) {
+	              var result = { label: data[i].name, value: data[i].id};
+	              results.push(result);
+	            }
+	            response(results);
+	          });
+	        },
+	        minLength: 3,
+	        select: function( event, ui ) {
+	          event.preventDefault();
+	          jq(selectedInput).val(ui.item.label);
+	        },
+	        change: function (event, ui) {
+	          event.preventDefault();
+	          jq(selectedInput).val(ui.item.label);
+	          console.log(ui.item.label);
+	          jq.getJSON('${ ui.actionLink("patientdashboardapp", "ClinicalNotes", "getFormulationByDrugName") }',
+	            {
+	              "drugName": ui.item.label
+	            }
+	          ).success(function(data) {
+	            var formulations = jq.map(data, function (formulation) {
+                    console.log(formulation);
+	              return new Formulation({ id: formulation.id, label: formulation.name});
+	            });
+	            prescription.drug().formulationOpts(formulations);
+	          });
+
+              //fetch the frequenciesui.
+              jq.getJSON('${ui.actionLink("patientdashboardapp","ClinicalNotes","getFrequencies")}').success(function(data){
+                   console.log(data);
+                  var frequency = jq.map(data, function (frequency) {
+                      return new Frequency({id: frequency.id, label: frequency.name});
+                  });
+                  prescription.drug().frequencyOpts(frequency);
+                });
+
+	        },
+	        open: function() {
+	          jq( this ).removeClass( "ui-corner-all" ).addClass( "ui-corner-top" );
+	        },
+	        close: function() {
+	          jq( this ).removeClass( "ui-corner-top" ).addClass( "ui-corner-all" );
+	        }
+	      });
+	    });
+});
+</script>
+
+<style>
+.simple-form-ui section {
+   width: 74%;
+}
+.dialog input {
+    display: block;
+    margin: 5px 0;
+    color: #363463;
+    padding: 5px 0 5px 10px;
+    background-color: #FFF;
+    border: 1px solid #DDD;
+    width: 97%;
+}
+.symptom-container, .diagnosis-container, .procedure-container, .investigation-container {
+    border-top: darkgrey dashed 1px;
+    margin-top: 8px;
+}
+.qualifier-container {
+    margin-left: 20px;
+    background-color: white;
+    padding: 5px 0 5px 15px;
+    font-size: .95em;
+}
+.qualifier-container p.qualifier-field label, .outcomes-container p.outcome {
+    font-size: .95em;
+    line-height: 19px;
+}
+.qualifier-option {
+    margin-bottom: 10px;
+}
+.pointer :hover {
+    cursor: pointer;
+}
+</style>
+>>>>>>> Allows user to add more than one drug- bug fix
```

Comment:
now... was woking on the sending patient back to the opd queue, was told it was more urgent. so I'm on it...